docs/generate_docs.bash

```
@@ -139,10 +139,16 @@ if [[ ${FAST_BUILD} = false ]]; then
     wait
 fi
 
-echo "Building docs"
+echo "Building HTML docs"
 # sphinx-build [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
 sphinx-build -M html source build --jobs auto $SPHINXOPTS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)